### PR TITLE
Increase modularization and fix layout

### PR DESCRIPTION
**Pull request by avaidyam**

### CODE REVIEW

File: header.php [hunk 1]
```diff
@@ -44,23 +44,22 @@
                     <span class="icon-bar"></span>
                     <span class="icon-bar"></span>
                 </button>
-                <?php if ($display_back_button) { ?>
+                <?php if (!$root) { ?>
                 <a href="/"><button type="button" class="navbar-left btn btn-link btn-dark" style="height: 50px; padding: 0px 15px; font-size: 18px; line-height: 20px; float: left;" aria-label="Back">
                     <i class="material-icons" style="color:white; font-size: 32px; margin-top: 4px;">arrow_back</i>
                 </button></a>
+                <?php } else { ?>
+                    <div class="navbar-left" style="height: 50px; width: 64px; padding: 0px 0px; float: left;"></div>
```
Comment (mattmolo) [inline header.php]: The padding on this actually looks kind of bad. If you're going to do it, add a 'house' icon where the back button would be, so it looks consistent.

File: join.php [hunk 2]
```diff
@@ -83,4 +88,24 @@ function clearEntries() {
         </div>
     </div>
 
+    <div class="well">
+        <div class="row">
+            <div class="col-lg-12 text-dark">
```
Comment (mattmolo) [inline join.php]: `<div class="col-md-8 col-md-offset-2 text-dark">` as in #53 to be consistent with above page

File: header.php (hunk 1 above)
Comment (mattmolo) [reply header.php]: I guess it could be the hamburger menu icon for the siderbar. maybe

File: officers/2016.php [hunk 3]
```diff
@@ -0,0 +1,233 @@
...
+    $site_title = 'Purdue IEEE';
+    $site_index = '/';
+    $root = true;
+    include '../header.php';
+?>
+
+    <div class="well text-dark">
+
+        <!--- Pagination -->
+        <div class="row text-center">
```
Comment (mattmolo) [inline officers/2016.php:12]: It makes it more visible, but this is also a pretty bad continuity thing from the other pages...

File: header.php (hunk 1 above)
Comment (avaidyam) [reply header.php]: [Using this one for now.](https://design.google.com/icons/#ic_apps)

File: officers/2016.php (hunk 3 above)
Comment (avaidyam) [reply officers/2016.php:12]: For now, this PR doesn't worry about the Officers page. I'll need your help to fix that up later.

File: header.php (hunk 1 above)
Comment (mattmolo) [reply header.php]: there's literally one called home: https://design.google.com/icons/#ic_home

File: header.php [hunk 4]
```diff
@@ -44,23 +44,24 @@
                     <span class="icon-bar"></span>
                     <span class="icon-bar"></span>
                 </button>
-                <?php if ($display_back_button) { ?>
+                <?php if (!$root) { ?>
                 <a href="/"><button type="button" class="navbar-left btn btn-link btn-dark" style="height: 50px; padding: 0px 15px; font-size: 18px; line-height: 20px; float: left;" aria-label="Back">
```
Comment (mattmolo) [inline header.php]: These should not be buttons..

Comment (avaidyam) [reply header.php]: But they are. You kinda click them...

Comment (mattmolo) [reply header.php]: screen shot 2016-09-22 at 15 16 36

Comment (mattmolo) [reply header.php]: That's what happens with a button

**Comment by mattmolo** [comment]
Resolves #58 #57 #54 #53

**Review by mattmolo — APPROVED** [review]

**Pull request merged**